docs/01_cva6_user/PMA.rst

```diff
@@ -1,24 +1,64 @@
-﻿..
-   Copyright (c) 2023 OpenHW Group
-   Copyright (c) 2023 Thales DIS design services SAS
-
-   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
-
-.. Level 1
-   =======
-
-   Level 2
-   -------
-
-   Level 3
-   ~~~~~~~
-
-   Level 4
-   ^^^^^^^
-
-.. _cva6_pma:
-
-PMA
-===
-Gap for step1 verification. Reuse DVplan from CV32E40S? Mike will reach out to some people who could help.
-
+﻿..
+   Copyright (c) 2023 OpenHW Group
+   Copyright (c) 2023 Thales DIS design services SAS
+
+   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+
+.. Level 1
+   =======
+
+   Level 2
+   -------
+
+   Level 3
+   ~~~~~~~
+
+   Level 4
+   ^^^^^^^
+
+.. _cva6_pma:
+
+PMA
+===
+
+An underlying system can have multiple physical memory attributes (PMA). CVA6
+supports three main access properties:
+
+- Non-idempotent regions (I/O regions): Regions marked as non-idempotent will
+  not be queried speculatively. Typical registers of that kind would be the UART
```

Comment:
I guess "will not be queried speculatively" could be made more explicit for the average user.